crates/apps_lib/src/client/rpc.rs

```diff
@@ -90,17 +91,30 @@ pub async fn query_and_print_masp_epoch(context: &impl Namada) -> MaspEpoch {
 /// Query and print some information to help discern when the next epoch will
 /// begin.
 pub async fn query_and_print_next_epoch_info(context: &impl Namada) {
-    query_block(context).await;
+    println!();
+    let current_time = query_block(context).await.unwrap();
```

Comment:
this is not exactly current time, but it's probably close enough. Still, maybe it would be better to just use `DateTimeUtc::now()` here

Reply:
Blocks are quick enough that I don't think there would be that much of a difference with `DateTimeUtc::now()`, especially considering the latency between reading the data and when it gets output to the user at CLI.

Looks like this got merged though, so I'm open to another PR that improves this.